corehq/form_processor/submission_post.py

```diff
@@ -156,6 +156,7 @@ def _get_success_message(self, instance, cases=None):
             user = CouchUser.get(instance.user_id)
             if not user or not user.is_web_user():
                 return _('Form successfully saved!')
+            user = WebUser.get(instance.user_id)
```

Comment:
instead of using `CouchUser.get` above, you can use `CouchUser.get_by_user_id` which will automatically wrap the doc correctly so you can avoid this second lookup

Reply:
thanks, updated